aea/cli/fetch.py

```diff
@@ -137,6 +137,10 @@ def fetch_agent_locally(
         )
 
     # add dependencies
+    if not ctx.config.get("is_local") or not ctx.config.get("is_mixed"):
+        # used by add_item to download locally
+        ctx.config["is_local"] = True
```

Comment:
By definition we are in the local case here; so if anything this should be set at the start of the method. But preferably; this is correctly set outside this method and then only asserted at the start.

Reply:
yes. If not necessary, I'd put this outside and put the asserts at the start of the function

Reply:
updated, check is added